src/views/main.html

```diff
@@ -1,19 +1,16 @@
 <!DOCTYPE html>
 <html>
-
-<head>
-  <title id="app-title">Bird Brain</title>
-  <link rel="stylesheet" href="../styles/main.css">
-</head>
-
-<body>
-  <br><br>
-  <img id="character" src="../images/pigeon1.png" height="200px;" />
-  <div style="width:300px;height:30px;background-color:#FF00FF">
-    <div id="volume" style="height:30px;background-color:#00FFFF"></div>
-  </div>
-  <div id="vol-num"></div>
-</body>
-<script src="../app/index.js"></script>
-
-</html>
\ No newline at end of file
+  <head>
+    <title id="app-title">Bird Brain</title>
+    <link rel="stylesheet" href="../styles/main.css">
+    <script defer src="../app/index.js"></script>
```

Comment:
`defer` means this code will not be loaded or executed until after the rest of the page loads.